docs/index.rst

```diff
@@ -176,3 +176,4 @@ Index
 * :ref:`search`
 
 .. _feedback@astropy.org: mailto:feedback@astropy.org
+.. _affiliated packages: http://www.astropy.org/affiliated/
```

Comment:
You HTTPS-ed everything else, why not here too? 😉 
```suggestion
.. _affiliated packages: https://www.astropy.org/affiliated/
```

Reply:
That's right! I amended the last commit.